content/en/docs/collector/transforming-telemetry.md

```diff
@@ -169,6 +169,43 @@ processors:
   k8sattributes/default:
 ```
 
+## Set status for span
+
+**Processor**:
+[span processor](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/processor/spanprocessor)
+
+The
+[span processor](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/processor/spanprocessor)
+can be used to change span status.
+
+The following setting is required:
+
+- `code:` Represents span status. One of the following values `Unset`, `Error`,
+  `Ok`.
+- `description:` setting is allowed only for code `Error`.
+
+Example:
+
+```yaml
+# Set status allows to set specific status for a given span. Possible values are
+# Ok, Error and Unset as per
+# https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/api.md#set-status
```

Comment:
All of this descriptive prose should be part of the main text, outside of the YAML.
@cartermp et al., WDYT?

Also, for the link use https://opentelemetry.io/docs/specs/otel/trace/api/#set-status if you embed it as a comment, otherwise use `/docs/specs/otel/trace/api/#set-status`

```suggestion
# Ok, Error and Unset as per https://opentelemetry.io/docs/specs/otel/trace/api/#set-status
```

Reply:
Eh, I kind of agree, but it's also helpful because if someone just copy-pastes this content, they will see a helpful comment in their config.

Reply:
I tend to like having comments in the YAML config snippets, as it's often where the eyes of our readers will go first. But I also like having the same content in the docs, to have some redundancy.

Reply:
hi @cartermp , @theletterf  do we need to duplicate the content outside the YAML block?

Reply:
None of the other YAML excerpts have such comments. I'd vote to put it all in the prose, and after, if necessary, we can see which snippet might make sense to include as a YAML comment. WDYT @cartermp and @theletterf?